eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/TranslationCollectorPass.php

```diff
@@ -45,7 +44,10 @@ public function process(ContainerBuilder $container)
         $definition = $container->getDefinition('translator.default');
         $collector = new GlobCollector($container->getParameterBag()->get('kernel.root_dir'));
 
-        $availableTranslations = [];
+        $availableTranslations = [
```

Comment:
Nitpick: I'd have used something like `$availableTranslations = [self::ORIGINAL_TRANSLATION];`.
You can then avoid the inline comment, and comment on the constant.